kotsadm/pkg/apiserver/bootstrap.go

```diff
@@ -106,6 +102,25 @@ func bootstrapIdentity() error {
 		}
 		identityConfig := obj.(*kotsv1beta1.IdentityConfig)
 
+		installationFile := filepath.Join(currentArchivePath, "upstream", "userdata", "installation.yaml")
+		installationData, err := ioutil.ReadFile(installationFile)
```

Comment:
this can be used instead
```
installation, err := kotsutil.LoadInstallationFromPath(filepath.Join(appDir, "upstream", "userdata", "installation.yaml"))
if err != nil {
	return "", errors.Wrap(err, "failed to load installation from path")
}
```